utils/docs/lepton-tragesym.1.in

```diff
@@ -39,12 +196,11 @@ scratch in Scheme by Vladimir Zhbanov <vzhbanov@gmail.com>.
 
 .SH SEE ALSO
 .BR lepton-schematic (1),
-.BR pcb (1)
 
 .SH COPYRIGHT
 .nf
 Copyright \(co 2012-2017 gEDA Contributors.
-Copyright \(co 2019 Lepton EDA Contributors.
+Copyright \(co 2019-2020 Lepton EDA Contributors.
```

Comment:
In other man pages we use a placeholder in copyright years
(e.g. `2019-@YEAR@`). Shouldn't it be used here?